Interfaces/IUniqueFilename.cs

```diff
@@ -0,0 +1,15 @@
+﻿using MyConsoleApp.UniqueFilenameClass;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MyConsoleApp.Interfaces
+{
+    public interface IUniqueFilename
+    {
```

Comment:
Naming improvement for interface - what is the purpose of a class that implements this interface? "Use Intention-Revealing Names" as Clean Code suggests.
Naming for method LogUniqueFilename is better, but maybe improvable
GenerateUniqueFilename is very good - I'd know what this is supposed to do!